_pages/schedule.md

```diff
@@ -4,4 +4,14 @@ title: Schedule
 permalink: /schedule/
 ---
 
-The StarCon 2019 schedule will be announced after our CFP closes.
+### Speakers
```

Comment:
can we take out this section to make this page more minimal? we already have a link in the navbar